tests/test_python_library.py

```diff
@@ -126,3 +128,47 @@ def test_split_system_tests():
         with open(templated_files / ".kokoro/presubmit/system-3.8.cfg", "r") as f:
             contents = f.read()
             assert "system-3.8" in contents
+
+
+def test_detect_versions_src():
+    temp_dir = Path(tempfile.mkdtemp())
+    src_dir = temp_dir / "src"
+    for v in ("v1", "v2", "v3"):
+        os.makedirs(src_dir / v)
+
+    with util.chdir(temp_dir):
+        versions = python.detect_versions()
+        assert ["v1", "v2", "v3"] == versions
+
+
+def test_detect_versions_staging():
+    temp_dir = Path(tempfile.mkdtemp())
+    staging_dir = temp_dir / "owl-bot-staging"
+    for v in ("v1", "v2", "v3"):
+        os.makedirs(staging_dir / v)
+
+    versions = python.detect_versions(staging_dir)
+    assert ["v1", "v2", "v3"] == versions
+
```

Comment:
Is the idea that the versions get detected from `owl-bot-staging`?

Reply:
Correct. I've renamed this test to `test_detect_versions_default_path` now that the default path is `owl-bot-staging`

Reply:
My mistake, the default path should be google/cloud. I've fixed this in 9a6f82f.